components/config-provider/__tests__/__snapshots__/components.test.js.snap

```diff
@@ -25192,8 +25192,8 @@ exports[`ConfigProvider components Tabs configProvider 1`] = `
         style="transform:translate(0px, 0px)"
       >
         <button
-          aria-selected="false"
-          class="config-tabs-tab"
+          aria-selected="true"
```

Comment:
The CI broken. I think there is no reason to do this change. Please make sure your branch checkout from the antd's latest feature branch.

And delete your old `node_modules` folder and reinstall it, then you can retry npm test and fix the issue.

Reply:
okay i will do it